docs/source/user/installing.rst

```diff
@@ -152,6 +152,8 @@ Then you can build and install Numba from the top level of the source tree::
 
     $ python setup.py install
 
+If you wish to run the test suite, see the :doc:`developer documentation <../developer/contributing>`.
```

Comment:
Please could this be line wrapped to 80 chars?

Also, if there's no anchor creating a link to the "how to run the test suite" part of the contributing docs, then perhaps it'd be a good idea to add one in there so there's a direct link that is available that can be easily shared (it also would mean this reference could jump straight to the relevant part of the docs).

Reply:
Thanks, I see in 79c2b3a there's no specific ref/link pair. An example of this for e.g. references and linking to the "continuous integration testing" section is:
**anchor**:
https://github.com/numba/numba/blob/061f21a73a574ea1b3451910a36db4f9b3ae788b/docs/source/developer/contributing.rst?plain=1#L325

**reference**:
https://github.com/numba/numba/blob/061f21a73a574ea1b3451910a36db4f9b3ae788b/docs/source/developer/contributing.rst?plain=1#L75

Reply:
Sorry, didn't get pushed.